catalog/ui/catalog-ui-search/src/main/webapp/component/workspace-item/workspace-item.less

```diff
@@ -1,141 +1,112 @@
-@{customElementNamespace}workspace-item.as-grid {
+@{customElementNamespace}workspace-item {
   .customElement;
+  overflow: hidden;
   display: inline-block;
-  width: auto;
+  width: 18rem;
```

Comment:
It looks fine.  Haven't had a chance to go back and update some of the px based margins, but they work well under zoom as is.